src/Makefile.am

```diff
@@ -53,7 +53,25 @@ gnome_games_SOURCES = \
 	dummy/dummy-icon.vala \
 	dummy/dummy-runner.vala \
 	\
+	gamepad/raw-gamepad-monitor.vala \
+	gamepad/raw-gamepad.vala \
+	gamepad/linux/guid-helpers.vala \
+	gamepad/linux/raw-gamepad-monitor.vala \
```

Comment:
The file should be named `linux-raw-gamepad-monitor.vala` to match the name of the main type it contains.